draft-ietf-rats-corim.md

```diff
@@ -2407,6 +2409,21 @@ If key verification succeeds for any _key_:
 
 Otherwise, do not add the `addition` ECT to the ACS.
 
+#### Copying instance field from a condition {#sec-comid-instance-copy}
+
+A CoRIM author may need to create a conditional endorsement which applies to all devices of a particular type, regardless of their instance field.
+If the first `stateless-environment` in the `conditions` field does not contain an `instance` codepoint then it will match against any environment with the same `class`.
```

Comment:
```suggestion
If the first `stateless-environment` in the `conditions` field does not contain an `instance` codepoint, then it will match against any environment with the same `class`.
```
This is roughly already the case, modulo `class` being `class` and/or `group`.

Reply:
> `stateless-environment`

This isn't in the cddl. Possibly it should say "environment"?

In general, matching applies to all the environment fields. If there is a class, then it will match the class even if there is also an instance or group. The interesting case is if there isn't a class but you want the addition to apply to a class of ECTs.